zerver/openapi/zulip.yaml

```diff
@@ -646,6 +756,45 @@ paths:
                                     },
                                   "id": 0,
                                 }
+                            - type: object
+                              description: |
+                                Event sent to all users of the Zulip server for
+                                removal of a user to the Zulip server.
```

Comment:
"removal of a user from the Zulip organization" would be proper English.  But rewrote this to refer to deactivating users, since we don't actually delete them with this event.